### ISSUE TITLE
Feature Request: Alias Name for Shortened URL

### ISSUE THREAD

**Issue by rrottmann** [robherley/snips.sh#174]
### What is the feature or enhancement?

As a user of the snips.sh, I would like the ability to associate an alias name with the shortened URL of my pasted content. This would allow me to easily update the content without having to modify the URL itself.

For example, if I were to paste some data related to an issue and receive a shortened URL of "f:yimGfTTYcl@snips.sh", I could then associate a alias name such as "a:issue123@snips.sh" with that URL. If I needed to update the data related to that issue, I could simply paste the new data and update the content associated with the alias.

Another example: When pasting the output of an async task in a shell script, one could register an alias and when the results are present, update the content.

Also when building a FAQ, having stable links pointing to updated pastes of e.g. command output would be benefitial. Pastes could then be updated without modifying links in the FAQ.

To sum it up: by providing the ability to associate an alias with the shortened URL, snips.sh would become more flexible and user-friendly, allowing users to easily manage and update their pasted content.

**Comment by robherley** [comment]
👋 @rrottmann apologies for the delay, but this now available in [v0.12.0](https://github.com/robherley/snips.sh/releases/tag/v0.12.0)

on upload you can specify a name:

```
cat main.go | ssh snips.sh -- -name main.go
```

and then it can be referenced like:

```
ssh n:main.go@snips.sh
```

or via the UI: https://snips.sh/f/H5SPi6aGrY/n/main.go

also, updating a snippet had been available since [v0.10.0](https://github.com/robherley/snips.sh/releases/tag/v0.10.0), which also works with these new named aliases:

```
echo "updated content" | ssh n:main.go:content@snips.sh
```
